src/auth-providers/better-auth/files/scripts/setup-db.mjs

```
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+// Apply sql/*.sql to DATABASE_URL in lexical order. No psql dependency.
+import { readFileSync, readdirSync } from 'node:fs';
+import { join, dirname } from 'node:path';
+import { fileURLToPath } from 'node:url';
+import pg from 'pg';
+
+const here = dirname(fileURLToPath(import.meta.url));
+const sqlDir = join(here, '..', 'sql');
+
+const url = process.env.DATABASE_URL;
```

ESLint

[error] 'process' is not defined.

(no-undef)